src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -14,6 +14,7 @@
 using System.Management.Automation.Internal;
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-windows`
method conversion.Correctly casts 100n to value 100 as type [int]
```powershell
Expected '100', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Conversions.Tests.ps1: line 510
510:         $Result | Should -Be $Value
```
method conversion.Correctly casts 1234s to value 1234 as type [bigint]
```powershell
Expected '1234', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Conversions.Tests.ps1: line 510
510:         $Result | Should -Be $Value
```